docs/index.rst

```diff
@@ -12,12 +12,124 @@ COSMOS Academy
    :alt: cosmos-academy-logo
    :align: center
 
+Welcome to the Cosmos Academy! This page will orient you around the Tendermint/Cosmos tech stack. It is roughly organized from beginner => expert with higher priority to content most relevant to many users.
 
-Contents
+Introduction
+============
+
+New here? Walk through some introductory content to get up to speed.
+
+.. toctree::
+   :maxdepth: 2
+
+   content/introduction.rst
+
+Testnets
+========
+
+Either create you own (a good first step) or join an existing testnet.
+
+.. toctree::
+   :maxdepth: 2
+
+   content/testnets.rst
+
+App Development
+===============
+
+Apps can be built either using the SDK or in any language for which an ABCI server exists.
+
+.. toctree::
+   :maxdepth: 2
+
+   content/app-development.rst
+
+Read The Docs
+=============
+
+Several core repositories have their tutorials and documentation versioned directly in the repo then built and hosted by Read The Docs. Here you get an overview of our core documentation.
+
+.. toctree::
+   :maxdepth: 2
+
+   content/read-the-docs.rst
+
+Tendermint & Cosmos
+===================
+
+Cosmos is powered by code from two GitHub organizations: Tendermint and Cosmos. Roughly, Tendermint contains the lower-level repositories that serve as building blocks for Cosmos projects.
+
+.. toctree::
+   :maxdepth: 2
+
+   content/tendermint-and-cosmos.rst
+
+Concepts
+========
+
+Things that require more explanation than the glossary can provide
+
+.. toctree::
+   :maxdepth: 2
+
+   content/concept.rst
+
+Glossary
 ========
 
+A bunch of words defined
+
 .. toctree::
-   :maxdepth: 4
+   :maxdepth: 2
+
+   content/glossary.rst
+
+Blog
+====
+
+A list of all articles published on the Medium blog.
```

Comment:
"A list of selected articles published on the Medium blog"

I don't think it's useful to link all articles. However, it's great to link the best articles in a logical order

Reply:
makes sense, link to medium so ppl can find all articles. Point of Cosmos Academy should be to curate the content